assets/app/views/settings.html

```diff
@@ -0,0 +1,83 @@
+<div ng-controller="ProjectController">
+  <project-page>
+    <div ng-controller="SettingsController">
+      <h1>Project Settings</h1>
+      <alerts alerts="alerts"></alerts>
+      <h2>General information</h2>
+      <!-- TODO make these fields editable - probably with a cog icon that flips into inline-editing -->
+      <dl class="dl-horizontal left">
+        <dt>Display name:</dt>
+        <dd ng-if="project.displayName">{{project.displayName}}</dd>
+        <dd ng-if="!project.displayName"><em>No display name</em></dd>
+        <dt>Description:</dt>
+        <dd ng-if="project.metadata.annotations && project.metadata.annotations.description">{{project.metadata.annotations.description}}</dd>
+        <dd ng-if="!project.metadata.annotations || !project.metadata.annotations.description"><em>No description</em></dd>
+      </dl>
+      <h2>
+        <span>Quota</span>
+        <a href="javascript:;" style="margin-left: 5px; cursor: pointer;" class="small pficon pficon-help" data-container="body" data-toggle="popover" data-placement="right" data-trigger="focus" data-content="Quota is enforced at the project level.  Once the maximum for the project has been reached no more of that resource can be allocated within the project."></a>
+      </h2>
+      <div ng-if="(quotas | hashSize) == 0"><em>{{emptyMessageQuotas}}</em></div>
+      <table ng-if="(quotas | hashSize) > 0" class="table">
+        <thead>
+          <th>Resource type</th>
+          <th>Used</th>
+          <th>Max</th>
+        </thead>
+        <tbody ng-repeat="quota in quotas">
+          <tr ng-if='!quota.status.used' class="danger">
+            <td colspan="5">
+              <span class="pficon-layered" data-toggle="tooltip" data-placement="right" title="Missing quota status" style="cursor: help;">
+                <span class="pficon pficon-error-octagon"></span>
+                <span class="pficon pficon-error-exclamation"></span>
+              </span>
+              Status has not been reported on this quota usage record.  Any resources limited by this quota record can not be allocated.
+            </td>
+          </tr>
+          <tr ng-class="{warning: quota.status.used && quota.status.used[resourceType] >= quota.status.hard[resourceType]}" ng-repeat="(resourceType, specMax) in quota.spec.hard">
```

Comment:
from ng-repeat="(resourceType, specMax) in quota.spec.hard   you can use variables set from a repeat on the same tag as the repeat